tests/sgx/thread_interrupt/enc/enc.c

```diff
@@ -312,13 +312,13 @@ void enc_thread_interrupt_blocking(int* ret)
 
     host_sleep_msec(30);
 
+    if (oe_sgx_td_register_host_signal(_thread_info_blocking.td, SIGUSR1) !=
```

Comment:
This change is unrelated to the PR description. Won't this silently skip the test?

Reply:
this is just for testing the pipeline failure. reverted the changes